Sources/ShopifyCheckoutSheetKit/CheckoutBridge.swift

```diff
@@ -44,12 +44,15 @@ enum CheckoutBridge: CheckoutBridgeProtocol {
         let colorScheme = ShopifyCheckoutSheetKit.configuration.colorScheme
         let platform = mapPlatform(ShopifyCheckoutSheetKit.configuration.platform)
 
-        return UserAgent.string(
+        let baseUserAgent = UserAgent.string(
             type: .standard,
             colorScheme: colorScheme,
             platform: platform,
             entryPoint: entryPoint
         )
+
+        // Include Chrome version to meet Google Pay minimum requirements (Chrome >= 137)
+        return "\(baseUserAgent) Chrome/137.0.0.0 Safari/537.36"
```

Comment:
Yes, this is dangerous to do. We enforce Chrome 137+ because that version introduces some security features. With this change we're always claiming to have security features that may no really exist. 

If this is the reason of why Google Pay is not being rendered, it should be ok to not do any changes.